feed.json

```diff
@@ -1,87 +1,75 @@
-            "viewAllLinkUrl": "https://www.youtube.com/channel/UC8qPRh20PpuxBvxf0w2WrwQ",
-            "cards": [
-                {
-                    "image": "https://via.placeholder.com/244x95/0078d4/ffffff?text=Bot+FrameWork+Composer",
-                    "title": "Introduction to Composer",
-                    "description": "A five minute intro to Composer",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/300x200/",
-                    "title": "Build a weather bot",
-                    "description": "An end to end tutorial for creating a weather bot",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/244x55/",
-                    "title": "Using prompts",
-                    "description": "Discover how to use prompts to accept input from users",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/200x95/",
-                    "title": "Using LG",
-                    "description": "Explore Language Generation and how to generate dynamic responses",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                }
-            ]
-        },
-        {
-            "title": "Articles",
-            "cards": [
-                {
-                    "image": "https://via.placeholder.com/244x95/09f/fff.png",
-                    "title": "Introduction to Composer Document",
-                    "description": "A five minute intro to Composer",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/244x95/09f/fff.png",
-                    "title": "Build a weather bot",
-                    "description": "An end to end tutorial for creating a weather bot",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/244x95/09f/fff.png",
-                    "title": "Using prompts",
-                    "description": "Discover how to use prompts to accept input from users",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                },
-                {
-                    "image": "https://via.placeholder.com/244x95/09f/fff.png",
-                    "title": "Using LG",
-                    "description": "Explore Language Generation and how to generate dynamic responses",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                }
-            ]
-        }
-    ]
+	"version": 1,
+	"whatsNewLinks": [{
+		"title": "Bot Framework Composer 1.3",
```

Comment:
Latest composer version is 1.4, is this out of date ?

Reply:
We don't currently have a 1.4 blog post - it will be added to the feed once available.